frontend/integration-tests/tests/environment.scenario.ts

```diff
@@ -0,0 +1,115 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { safeLoad, safeDump } from 'js-yaml';
+import * as _ from 'lodash';
+
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import * as crudView from '../views/crud.view';
+import * as environmentView from '../views/environment.view';
+import * as yamlView from '../views/yaml.view';
+
+browser.ignoreSynchronization = true;
+
+const BROWSER_TIMEOUT = 15000;
+const WORKLOAD_NAME = `env-${testName}`;
+const Actions = {
+  add: 'add',
+  delete: 'delete',
+};
+
+describe('<Environment Page />', () => {
```

Comment:
The e2e tests should be framed as full _scenarios_, not component-level tests. So you could use:
```javascript
describe('Interacting with the environment variable editor', () => {
...
});
```

Reply:
Right!  I tend toward wanting something a little more verbose, describing exactly what the test is doing, so that when it breaks, the error message is clear for the poor dev who broke it 😄